flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+    "only-arches": ["x86_64"]
```

Comment:
this remain disappointing.

Reply:
got rid of this as well.

Reply:
you didn't understand. 

What's disappointing is that it's only for x86_64. Right now you actually broke the build because you still try to build something that doesn't run on aarch64 that isn't aarch64.

Reply:
Yeah i am bringing it back. The app runs only on x86_64 right now.

Reply:
still not